plugins/woocommerce/tests/e2e-pw/tests/activate-and-setup/complete-onboarding-wizard.spec.js

```diff
@@ -41,8 +45,8 @@ test.describe( 'Store owner can complete onboarding wizard', () => {
 
 		// Navigate back to "Store Details" section
 		await page.click( 'button >> text=Store Details' );
-
 		await onboarding.handleSaveChangesModal( page, { saveChanges: true } );
+		await page.locator( 'text="Welcome to WooCommerce"' ).waitFor();
```

Comment:
This ensures that the Store details page is loaded completely before navigating back to the Industry section. This fixes the flakiness in WP-Env localhost.